packages/vite/src/node/server/hmr.ts

```diff
@@ -165,25 +165,26 @@ export async function handleFileAddUnlink(
   server: ViteDevServer,
   isUnlink = false
 ) {
+  const mods = server.moduleGraph.getModulesByFile(file) ?? []
+  const modules = [...mods]
```

Comment:
**suggestion (non-blocking):** Seems `mods` is not needed below
We may merge these two lines just to 
```ts
const modules = [...server.moduleGraph.getModulesByFile(file)]
// or if this errors
const modules = [...(server.moduleGraph.getModulesByFile(file) ?? [])]
```

Reply:
It's also not needed on line 169 :thinking: 
So we may move it down again to line 171, should we?

Reply:
> It's also not needed on line 169 🤔
> So we may move it down again to line 171, should we?

`modules` is needed in 179

Reply:
> Seems mods is not needed below

Done.